internal/bitset/bitset.go

```diff
@@ -0,0 +1,90 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+// Third-party implementation: https://github.com/willf/bitset/blob/master/bitset.go
+
+// Package bitset implements a bitset.
```

Comment:
This isn't documentation.

Reply:
Do I need to remove it or add something?

Reply:
Sorry. I just meant that we should describe what a bitset is and what this package enables a user to do.